Cargo.lock

```diff
@@ -1330,7 +1281,7 @@ dependencies = [
 
 [[package]]
 name = "binary-merkle-tree"
-version = "4.0.0-dev"
+version = "10.0.0"
```

Comment:
where are all these changes coming from?

for complete crate compatibility on this patch release we shouldn't bump any dependencies references

Reply:
I don't really know, but the benchmark job needed it to not throw an error. All the changes are right here in this PR, so no idea. I can remove this file from here and hope it builds without the changes.